cg/services/parse_run_completion_status_service/parse_run_completion_status_service.py

```diff
@@ -0,0 +1,37 @@
+"""Service to parse the RunInfo.xml file."""
+
+import re
+from pathlib import Path
+
+from xml.etree.ElementTree import ElementTree
+
+from cg.constants.demultiplexing import RunCompletionStatusNodes
+from cg.io.xml import read_xml
+from datetime import datetime
+
+DATE_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+
+class ParseRunCompletionStatusService:
+
+    @classmethod
+    def format_time(cls, time: str) -> datetime:
+        """Format the time."""
+        no_microseconds: str = re.sub(r"\.\d+", "", time)
+        no_microseconds_no_time_zone = re.sub(r"[+-]\d{2}:\d{2}$", "", no_microseconds)
+        return datetime.strptime(no_microseconds_no_time_zone, DATE_TIME_FORMAT)
+
+    @staticmethod
+    def _parse_run_completion_status_path(run_completion_status_path: Path) -> ElementTree:
```

Comment:
Do we really need this function? It seems like a wrapper not encapsuling any complicated logic, maybe the `read_xml` can be called directly in the methods below. Let me know if this is not the case and feel free to disregard